tools/dicom-web-electron/package-lock.json

```diff
@@ -4757,9 +4757,9 @@
       }
     },
     "follow-redirects": {
-      "version": "1.14.4",
-      "resolved": "https://registry.npmjs.org/follow-redirects/-/follow-redirects-1.14.4.tgz",
-      "integrity": "sha512-zwGkiSXC1MUJG/qmeIFH2HBJx9u0V46QGUe3YR1fXG8bXQxq7fLj0RjLZQ5nubr9qNJUZrH+xUcwXEoXNpfS+g=="
+      "version": "1.14.7",
+      "resolved": "https://registry.npmjs.org/follow-redirects/-/follow-redirects-1.14.7.tgz",
+      "integrity": "sha512-Q6cWsys88HoPgAaFAVUb0WpPk0O8iTeisR9IMqy9G8AbO4NlpVknrnQS03zzF9PGAWgO3cgletO3VjV/P7VztA=="
```

Comment:
I would think this integrity should match the value found above with the `node_modules/follow-redirects` package. This looks like the integrity of the `axios` package.

Reply:
Looks like running `npm install` updates this value to match.

Reply:
Good catch! By mistake, I had copied the wrong one. It's fixed now.